assets/dev/scss/editor/preview/_section.scss

```
@@ -82,8 +82,7 @@
 
 // Top Section / Container (Handles inside)
 .e-handles-inside,
-.elementor-section--handles-inside,
-.elementor-edit-mode [data-element_type]:first-child {
+.elementor-section--handles-inside {
```

```suggestion
{
```

remove and use only the generic class?